packages/uswds-core/src/styles/mixins/helpers/checkbox-and-radio-colors.scss

```diff
@@ -16,6 +16,21 @@
   @include -checkbox-and-radio-colors($input-type: "radio", $args...);
 }
 
+@mixin selected-tile-colors($input-active-color) {
```

Comment:
## Mixin name

I used the name `selected` tile/input colors because both `checked` and `indeterminate` reflect a sort of selected state.

Let me know if there is a better name we should use here.